modules/cudacodec/src/video_reader.cpp

```diff
@@ -74,6 +74,9 @@ void videoDecPostProcessFrame(const GpuMat& decodedFrame, GpuMat& outFrame, int
         outFrame.create(height, width, CV_8UC1);
         cudaMemcpy2DAsync(outFrame.ptr(), outFrame.step, decodedFrame.ptr(), decodedFrame.step, width, height, cudaMemcpyDeviceToDevice, stream);
     }
+    else if (colorFormat == ColorFormat::YUV) {
+        decodedFrame.copyTo(outFrame);
```

Comment:
It should be done with stream too, I think.

Reply:
Indeed it should, thanks.

@asmorkalov are CUDA builds/tests not working at the moment?

Reply:
Yes. We have issues with build farm right now, and have to rework it significantly.